tests/resources/functions/python-3.10/tests.py

```diff
@@ -1,39 +1,68 @@
-import json
 import requests
+import os
 
-#    'req' variable has:
-#        'headers' - object with request headers
-#        'payload' - object with request body data
-#        'variables' - object with function variables
-#    'res' variable has:
-#        'send(text, status)' - function to return text response. Status code defaults to 200
-#        'json(obj, status)' - function to return JSON response. Status code defaults to 200
-#    
-#    If an error is thrown, a response with code 500 will be returned.
+def main(context):
+    action = context.req.headers.get('x-action', None)
 
-def main(req, res):
-    payload = json.loads('{}' if not req.payload else req.payload)
-    todo_id = payload.get('id', 1)
+    if action == 'plaintextResponse':
+        return context.res.send('Hello World 👋')
+    elif action == 'jsonResponse':
+        return context.res.json({ 'json': True, 'message': 'Developers are awesome.' })
+    elif action == 'redirectResponse':
+        return context.res.redirect('https://github.com/')
+    elif action == 'emptyResponse':
+        return context.res.empty()
+    elif action == 'noResponse':
+        context.res.send('This should be ignored, as it is not returned.')
+    elif action == 'doubleResponse':
+        context.res.send('This should be ignored.')
```

Comment:
Shouldn't this throw an error ( i.e. return a 500 response )

Reply:
First one does not return, so its ignored. THat is the rest.

Double return is not allowed in genreal. language wont let you

Reply:
okay